### PR TITLE
add responsive version for desktop

### PR DESCRIPTION
**Pull request by MarsIncarnate**
This PR add a responsive version for large screen sizes

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -379,3 +387,182 @@ form input[type=submit] {
...
+    grid-column-end: 4;
+  }
+
+  #about {
+    position: absolute;
+    top: 2411px;
+    left: 50%;
+    transform: translate(-50%);
+    width: 100%;
+  }
```
Comment (anagudelogu) [inline styles.css:497]: You did an awesome job working on your desktop version! 💯 However, there's still some room to improve in this section:

|Template|Your design|
|---|---|
| image | image |

- [ ] Please consider adding more horizontal spacing to the edges of this section, at the moment, the text overlaps with the background image! 🚧 

|Template|Your design|
|---|---|
| image | image |

- [ ] The "Get my resume" button overlaps your paragraph text in the mobile version. Please dig into this and fix it! 🚧

Please consider applying these changes! 🤓 🛠️

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @MarsIncarnate, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] **[OPTIONAL]:** I see that you use the `position: absolute` property on multiple elements across your stylesheet. I strongly suggest you avoid using this property if you don't need to, as it creates a separate document flow for your elements and will make them hard to lay out. You could read more about this [here](https://ishadeed.com/article/less-absolute-positioning-modern-css/).


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -379,3 +387,183 @@ form input[type=submit] {
...
```
Comment (franclobo) [inline styles.css:497]: - [ ] Nice work following the instructions of the last code reviewer and fixing the mobile version, but know the button is overlapping the paragraph in the desktop version, please 🙏🏽 fix it for a better user interface.
image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @MarsIncarnate, 

# Required Changes ♻️
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️
- Professional README file ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lizdev-05 — APPROVED** [review]
Hi @MarsIncarnate, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

- [ ] [OPTIONAL] Kindly check your responsiveness on 768 widths upward, the content overlap, 👇
image
image
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**